resources/com.sap.piper/pipeline/abapStageDefaults.yml

```diff
@@ -5,7 +5,9 @@ stages:
   Clone Repositories: {}
   ATC: {}
   Build: {}
-  Integration Tests: {}
+  Integration Tests:
+    confirmDeletion: false
   Confirm: {}
   Publish: {}
-  Post: {}
+  Post:
+    confirmDeletion: false
```

Comment:
-Option 1: set `confirmDeletion` to true and assume users want to confirm first before deleting the Integration System
-Option 2: set `confirmDeletion` to false for faster pipeline execution if it hasn't been configured in the stage configuration

I guess this really is a design question and depends on which option will be more used in the end. Both options are compatible whereas I would also prefer option 2 as a "faster" and more meaningful option.

Reply:
From previous discussions we had about the pipeline:

- For *Integration Tests* stage: The system should be deleted **after confirmation**, the common understanding was that a transient system is used for the add-on installation test. There is no good reason at the moment not to delete this system: patch tests (updating existing system instead of installation of add-on) will not be possible with 2011 as the Landscape Portal can only trigger updates to "released" target versions (and determination of version cannot be influenced by partner)
**Only if the Jenkins user does confirm successful installation of the add-on product, the system should be deleted.**
- For *Post* stage: The system should be deleted by default, the common understanding was here as well that we use a transient system. 
In principle the SAP_COM_0582 services should be repeatable under any circumstances, but we have some issues with the commit history for example (local history deleted during maintenance event and branching problem that is discussed in parallel). Therefore I would not promote in any way to reuse the assembly systems for builds. **If the pipeline execution fails (error ins stage previous to *Post*), the system should not be deleted.**